backend/src/main/java/com/softwareharddata/bbetter/exception/UserServiceExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.softwareharddata.bbetter.exception;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class UserServiceExceptionHandler {
+
+    private final static Logger LOGGER= LoggerFactory.getLogger(UserServiceExceptionHandler.class);
```

Comment:
you could also use the lombok annotation
https://projectlombok.org/features/log

than you can just use the logger